### ISSUE TITLE
Create GitHub Action to scan for broken ICO links

### ISSUE THREAD

**Issue by garethrees** [mysociety/whatdotheyknow-theme#1714]
We're often finding and updating broken links to the ICO website and it would be good to catch these sooner so that fewer users are left not knowing what to do.

I spiked out one approach in https://github.com/mysociety/whatdotheyknow-theme/issues/1112#issuecomment-1559685832.

For this to be effective we'd want it running periodically, and to notify us of any links that no longer work.

I expect we could run the script, then dump the output in a new issue if the exit code is non-zero.

**Comment by mdeuk** [comment]
+1.

This would certainly be handy, more generally, for external links on the help pages.

**Comment by garethrees** [comment]
Yeah, I had thought about broken links more generally, but intentionally wanted to limit the scope to ICO links to just get the routine up and running and then expand out from there.

**Comment by HelenWDTK** [comment]
This issue is being closed due to a lack of discussion or resolution for over 12 months. Should we decide to revisit this issue in the future, it can be reopened.

**Comment by garethrees** [comment]
Useful notes on this problem in https://github.com/openaustralia/righttoknow/issues/870#issuecomment-5216738938
